packages/core/components/ComponentList/index.tsx

```diff
@@ -39,7 +39,7 @@ const ComponentList = ({
   return (
     <div className={getClassName({ isExpanded: expanded })}>
       {title && (
-        <div
+        <button
```

Comment:
I wonder if these items should just be removed from the tab index for the time-being as they're not functional

Reply:
Bit confused on this one, as I can now tab to the buttons and toggle the section below open/closed with the keyboard.

Reply:
Sorry, I meant to comment on the component item themselves, not the group title.

The component items are not draggable by keyboard, so I wonder if we should remove them from the tab index.

Reply:
Cool, agree, let's close!